app/src/app/navtitle/navtitle.component.html

```diff
@@ -3,33 +3,61 @@ <h1 i18n><a routerLink="/">Itero</a></h1>
       <span class="navitem" i18n>Log in</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged"  routerLink="/r/create" routerLinkActive="active">
+    <a *ngIf="(session.state$ | async).logged" routerLink="/r/create" routerLinkActive="active" id="navitembig">
       <span class="navitem">Create</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged" [matMenuTriggerFor]="profileMenu">
+    <a [matMenuTriggerFor]="profileMenu" id="naviconmenu">
+      <span class="navitem"><mat-icon>menu</mat-icon></span>
+    </a>
+    <a *ngIf="(session.state$ | async).logged" [matMenuTriggerFor]="profileMenu" id="naviconperson">
       <span class="navitem"><mat-icon>person</mat-icon></span>
     </a>
   </nav>
 </div>
 <mat-menu #profileMenu="matMenu" class="nav-menu" xPosition="before">
-  <button mat-menu-item (click)="notify.askAUthorization()" [disabled]="(notify.permission$ | async) != 'default'">
+  <button *ngIf="(demoIsActive$ | async) && !(session.state$ | async).logged" mat-menu-item id="navitemsmall"><mat-icon>poll</mat-icon>
+    <a [routerLink]="(demoPath$ | async)" routerLinkActive="active">
+      <span class="navitem" i18n>Try it</span>
+    </a>
+  </button>
+  <button *ngIf="!(session.state$ | async).logged" mat-menu-item id="navitemsmall"><mat-icon>person_add</mat-icon>
+    <a routerLink="/r/session/signup" routerLinkActive="active">
+      <span class="navitem" i18n>Sign up</span>
+    </a>
+  </button>
+  <button *ngIf="!(session.state$ | async).logged" mat-menu-item id="navitemsmall"><mat-icon>person_outline</mat-icon>
+    <a routerLink="/r/session/login" routerLinkActive="active">
+      <span class="navitem" i18n>Log in</span>
+    </a>
+  </button>
+  <button *ngIf="(session.state$ | async).logged" mat-menu-item id="navitemsmall"><mat-icon>poll</mat-icon>
+    <a routerLink="/r/list" routerLinkActive="active">
+      <span class="navitem" i18n> Polls</span>
+    </a>
+  </button>
+  <button *ngIf="(session.state$ | async).logged" mat-menu-item id="navitemsmall"><mat-icon>add_box</mat-icon>
+    <a routerLink="/r/create" routerLinkActive="active">
+      <span class="navitem"> Create</span>
+  </a></button>
+  <hr *ngIf="(session.state$ | async).logged">
+  <button *ngIf="(session.state$ | async).logged" mat-menu-item (click)="notify.askAUthorization()" [disabled]="(notify.permission$ | async) != 'default'">
     <ng-container [ngSwitch]="notify.permission$ | async">
       <mat-icon *ngSwitchCase="'default'">notifications</mat-icon>
       <mat-icon *ngSwitchCase="'denied'">notifications_off</mat-icon>
       <mat-icon *ngSwitchCase="'granted'">notifications_active</mat-icon>
     </ng-container> Notifications
   </button>
-  <button mat-menu-item (click)="logoff()"><mat-icon>logout</mat-icon> Log off</button>
+  <button *ngIf="(session.state$ | async).logged" mat-menu-item (click)="logoff()"><mat-icon>logout</mat-icon> Log off</button>
```

Comment:
This part is the "profile menu". You could make it visible by enclosing it inside an `<ng-container>`. It would allow you to have only one ngIf. ng-container are removed during compilation, hence it will not appear on the client page.